description: use __alignof__ since as of gcc8 alignof returns 4 != 8 bytes on i386
origin: https://src.fedoraproject.org/cgit/rpms/chromium.git/tree/chromium-66.0.3359.170-gcc8-alignof.patch
author: Fedora chromium maintainers

--- a/mojo/public/c/system/macros.h
+++ b/mojo/public/c/system/macros.h
@@ -28,7 +28,11 @@
 // clang-format on
 
 // Like the C++11 |alignof| operator.
+#if defined(__GNUC__)
+#define MOJO_ALIGNOF(type) __alignof__(type)
+#else
 #define MOJO_ALIGNOF(type) alignof(type)
+#endif
 
 // Provides a convenient test for the presence of a field in a user-provided
 // structure from a potentially older version of the ABI. Presence is determined
